packages/@core.next/auth.jsx

```diff
@@ -28,7 +28,7 @@ const userFragment = `
   isAdmin
 `
 
-let USER_QUERY = gql`
+export let USER_QUERY = gql`
```

Comment:
This variable is not a part of a public API. You should pass an arguments for `useAuth`